### ISSUE TITLE
[V2] Access is allowed from event dispatch thread only.

### ISSUE THREAD

**Issue by olivernybroe** [jimkyndemeyer/js-graphql-intellij-plugin#256]
## Version and Environment Details
__Operation system__: MacOS Mojava Version 10.14.4

__IDE name and version__: 
PhpStorm 2019.2 EAP
Build #PS-192.4205.18, built on May 22, 2019

__Plugin version__: 2.1.0

## Description
It spams me with errors all the time, but it seems to be working fine though.
I have created a gist wit the 3 error files so you can review them and debug through them
https://gist.github.com/olivernybroe/1a7f568bd0974513bdedd0508c900587

**Comment by jimkyndemeyer** [comment]
Ho Oliver.

Thanks for using the plugin.

I took a look at the gists. I should be able to reproduce and fix the issue on that basis.

Best regards,
Jim.

**Comment by blacksmoke26** [comment]
Same fatal error:

```
PhpStorm 2019.2 EAP
Build #PS-192.4205.18, built on May 22, 2019
PhpStorm EAP User
Runtime version: 11.0.3+12-b262.1 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 8.1 6.3
```

**Comment by jimkyndemeyer** [comment]
Fixed in the upcoming 2.1.1 release.

**Comment by dan-lee** [comment]
@jimkyndemeyer Thank you for fixing this! Could you release the new version soon? My IDE is spammed with errors and your wonderful extension doesn't work anymore 😢

**Comment by olivernybroe** [comment]
@dan-lee You can always just build it yourself and install the zip. Normally with intellij plugins you can just run the gradle command `buildPlugin` and then you get the zip.

**Comment by jimkyndemeyer** [comment]
I hope to release it by the end of the week. You can build it like Oliver said, but the plugin should also work in the stable non-EAP webstorm.

**Comment by jimkyndemeyer** [comment]
Testing with the EAP I found other new assertions introduced by https://github.com/JetBrains/intellij-community/commit/a0c101cd996f59b93d14f095b66b133892783cb9

This breaks the error annotator in the EAP, so please downgrade to the latest stable version if you encounter problems.

This is holding back the 2.1.1 release until I can work out how to get the editor instance without triggering the exception.

**Comment by jimkyndemeyer** [comment]
Fixed in the 2.1.1 release, pending JetBrains approval.

**Comment by olivernybroe** [comment]
Amazing work! Thanks a lot, looking forward to the new release :)
